packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Added optional method `getDsDocumentFilePath` to `IZoweDatasetTreeNode` interface.
```

Comment:
We should include an issue/pr link for this. Also, we should explain why we made this change and/or the benefit of this change to the user.